src/LDX/VoteReward/Main.php

```diff
@@ -156,4 +158,4 @@ public function rewardPlayer($player, $multiplier) {
         $player->sendMessage("[VoteReward] You voted on $multiplier server list" . ($multiplier == 1 ? "" : "s") . "!");
     }
 
-}
\ No newline at end of file
+}
```

Comment:
Perhaps you should remove the newline you added at the end of the file since it wasn't there previously